### PR TITLE
Update complex_data_manipulation Example uptime in days/hours/minutes/seconds

### PR DESCRIPTION
**Pull request by fbourqui**
##### SUMMARY
Timedelta {{ now() - now().fromtimestamp(now(fmt='%s') | int - ansible_uptime_seconds) }}

Example gives uptime with microsecond precision with an original value in seconds: ansible_uptime_seconds

    "msg": [                                                                                                                                                                                                                                                                
        "Timedelta 15 days, 3:42:40.680283"                                                                                                                                      
    ]   

those microseconds are comming from {{ now() - now().fromtimestamp(now(fmt='%s') | int ) }}

Timedelta {{ now().replace(microsecond=0) - now().fromtimestamp(now(fmt='%s') | int - ansible_uptime_seconds) }}
With the change, you will get:

    "msg": [                                                                                                                                                                                                                                                                
        "Timedelta 15 days, 3:42:40"                                                                                                                                      
    ]   

##### ISSUE TYPE
- Docs Pull Request

+label: docsite_pr

##### SUMMARY
<!--- Describe the change below, including rationale and design decisions -->

<!--- HINT: Include "Fixes #nnn" if you are fixing an existing issue -->

##### ISSUE TYPE
<!--- Pick one below and delete the rest -->
- Bugfix Pull Request
- Docs Pull Request
- Feature Pull Request
- Test Pull Request

##### COMPONENT NAME
<!--- Write the short name of the module, plugin, task or feature below -->

##### ADDITIONAL INFORMATION
<!--- Include additional information to help people understand the change here -->
<!--- A step-by-step reproduction of the problem is helpful if there is no related issue -->

<!--- Paste verbatim command output below, e.g. before and after your change -->
```paste below

```

### CODE REVIEW

File: docs/docsite/rst/user_guide/complex_data_manipulation.rst [hunk 1]
```diff
@@ -294,14 +294,14 @@ An example on how to use facts to find a host's data that meets condition X:
     uptime_of_host_most_recently_rebooted: "{{ansible_play_hosts_all | map('extract', hostvars, 'ansible_uptime_seconds') | sort | first}}"
 
 
-Using an example from @zoradache on reddit, to show the 'uptime in days/hours/minutes' (assumes facts where gathered).
+Using an example from @zoradache on reddit, to show the 'uptime in days/hours/minutes/seconds' (assumes facts where gathered).
```
Comment (samccann) [inline docs/docsite/rst/user_guide/complex_data_manipulation.rst]: We should probably not put someone's name in the docs.

Comment (samccann) [inline docs/docsite/rst/user_guide/complex_data_manipulation.rst]:
```suggestion
Using an example from on `reddit <https://www.reddit.com/r/ansible/comments/gj5a93/trying_to_get_uptime_from_seconds/fqj2qr3/>`_, to show the 'uptime in days/hours/minutes/seconds' (assumes facts where gathered).
```

File: docs/docsite/rst/user_guide/complex_data_manipulation.rst [hunk 2]
```diff
@@ -294,14 +294,14 @@ An example on how to use facts to find a host's data that meets condition X:
...
 https://www.reddit.com/r/ansible/comments/gj5a93/trying_to_get_uptime_from_seconds/fqj2qr3/
```
Comment (samccann) [inline docs/docsite/rst/user_guide/complex_data_manipulation.rst]:
```suggestion
```

File: docs/docsite/rst/user_guide/complex_data_manipulation.rst (hunk 1 above)
Comment (fbourqui) [reply docs/docsite/rst/user_guide/complex_data_manipulation.rst]: The comment change is fine with me, it was present before, did not want to remove attribution.

**Comment by samccann** [comment]
Thanks @fbourqui and sorry if my edits messed up this PR.

**Pull request closed without merging**